MultiVersions/Forge/main/manyideas_core/lang/zh_tw.json

```
@@ -0,0 +1,155 @@
+						"block.manyideas_core.wood_stripped_smooth_warped": "平滑剝皮扭曲菌絲體",
+			"main_blocks_debug": "方塊：除錯",
+				"block.manyideas_core.debug_block_cullface": "Cullface 除錯方塊",
+			"main_blocks_dye_crafting_table": "方塊：染色台",
+				"block.manyideas_core.dye_crafting_table": "染色台",
+				"container.manyideas_core.dye_crafting_table": "染色台",
+			"main_blocks_end_block": "方塊：終界方塊",
+				"block.manyideas_core.end_block": "終界方塊",
+			"main_blocks_mortar": "方塊：研磨缽",
+				"block.manyideas_core.mortar": "研磨缽",
+			"main_blocks_table_saws": "方塊：桌鋸",
+				"main_blocks_table_saws_table_saw_diamond": "桌鋸：鑽石桌鋸",
+					"block.manyideas_core.table_saw_diamond": "鑽石桌鋸",
+					"container.manyideas_core.table_saw_diamond": "鑽石桌鋸",
+				"main_blocks_table_saws_table_saw_iron": "桌鋸：鐵桌鋸",
+					"block.manyideas_core.table_saw_iron": "鐵桌鋸",
+					"container.manyideas_core.table_saw_iron": "鐵桌鋸",
+				"main_blocks_table_saws_table_saw_stone": "桌鋸：石桌鋸",
+					"block.manyideas_core.table_saw_stone": "石桌鋸",
+					"container.manyideas_core.table_saw_stone": "石桌鋸",
+			"main_blocks_vanilla_blocks": "方塊：原版方塊",
+				"main_blocks_vanilla_blocks_flowers_straight": "原版方塊：直立花朵",
+					"main_blocks_vanilla_blocks_flowers_straight_normal": "直立花朵：小型花",
+						"block.manyideas_core.flower_straight_allium": "直立紫紅球花",
+						"block.manyideas_core.flower_straight_orchid_blue": "直立藍色蝴蝶蘭",
+					"main_blocks_vanilla_blocks_flowers_straight_tall": "直立花朵：大型花",
+						"block.manyideas_core.flower_tall_straight_lilac": "直立紫丁香",
+						"block.manyideas_core.flower_tall_straight_peony": "直立牡丹花",
+						"block.manyideas_core.flower_tall_straight_rose_bush": "直立玫瑰叢",
+						"block.manyideas_core.flower_tall_straight_sunflower": "直立向日葵",
+		"main_items": "物品",
+			"main_items_dyes": "物品：染料",
+				"item.manyideas_core.dye_rainbow": "彩虹染料",
+			"main_items_panels": "物品：板材",
+				"item.manyideas_core.panel_acacia": "相思木板",
+				"item.manyideas_core.panel_birch": "樺木板",
+				"item.manyideas_core.panel_crimson": "緋紅蕈木板",
+				"item.manyideas_core.panel_dark_oak": "黑橡木板",
+				"item.manyideas_core.panel_jungle": "叢林板",
+				"item.manyideas_core.panel_oak": "橡木板",
+				"item.manyideas_core.panel_spruce": "杉木板",
+				"item.manyideas_core.panel_warped": "扭曲蕈木板",
+			"main_items_plates": "物品：板材",
+				"item.manyideas_core.plate_copper": "銅板",
+				"item.manyideas_core.plate_gold": "金板",
+				"item.manyideas_core.plate_iron": "鐵板",
+				"item.manyideas_core.plate_quartz": "石英板",
+			"main_items_saws": "物品：桌鋸",
+				"item.manyideas_core.saw_diamond": "鑽石圓鋸",
+				"item.manyideas_core.saw_iron": "鐵圓鋸",
+				"item.manyideas_core.saw_stone": "石圓鋸",
+			"main_items_tools": "物品：工具",
+				"item.manyideas_core.mysterious_shears": "神秘剪刀",
+				"item.manyideas_core.redstone_key": "紅石鑰匙",
+		"main_messages": "訊息",
+			"main_messages_redstone_key": "訊息：紅石鑰匙",
+				"main_messages_redstone_key_opened_by": "紅石鑰匙：開門方式",
+					"main_messages_redstone_key_opened_by_both": "開門方式：雙種",

```

一般聽到的用法會是 `雙重` 或 `兩種`
```suggestion
					"main_messages_redstone_key_opened_by_both": "開門方式：雙重",

```